ckanext/stats/stats.py

```diff
@@ -116,6 +117,13 @@ def top_tags(cls, limit: int = 10,
 
     @classmethod
     def top_package_creators(cls, limit: int = 10) -> list[tuple[Optional[model.User], int]]:
+        if not config.get('ckan.auth.public_user_details'):
+            log.warning(
```

Comment:
```suggestion
            log.debug(
```

@avdata99 let's change the log verbosity to `debug`. This isn't a malfunction or something going wrong, it's just a configuration value so `debug` should be enough.